pkgs/servers/sql/postgresql/ext/pg_background.nix

```
@@ -0,0 +1,26 @@
+{
+  fetchFromGitHub,
+  lib,
+  postgresql,
+  postgresqlBuildExtension,
+}:
+
+postgresqlBuildExtension (finalAttrs: {
+  pname = "pg_background";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "vibhorkum";
+    repo = "pg_background";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9fW5wHdo9r5fLwU8zN2EEVSWxa+7q2qMjPpMo6iCavg=";
+  };
+
+  meta = {
+    description = "PRun PostgreSQL Commands in Background Workers";
+    homepage = "https://github.com/vibhorkum/pg_background";
```

Since upstream is using GitHub releases, we can add `meta.changelog` here, which will provide a link to it when auto-update PRs are created:


```suggestion
    homepage = "https://github.com/vibhorkum/pg_background";
    changelog = "https://github.com/vibhorkum/pg_background/releases/tag/v${finalAttrs.version}";
```